reference/JsCopyStringUtf16.md

```diff
@@ -29,3 +29,5 @@ When start is out of range or < 0, returns JsErrorInvalidArgument
 and `written` will be equal to 0. If calculated length is 0 (It can be due to string length or `start`
 and length combination), then `written` will be equal to 0 and call
 returns JsNoError.
+
+This API requires an active script context.
```

Comment:
Thanks for pointing it out @xiaoyinl . Both API require script context. Can you change this line (and the same line for JsCreateString) to 

> Requires an active script context.

So that it matches all the other API references ([example](https://github.com/Microsoft/ChakraCore/wiki/JsCreateObject)).